[2022-06-16 00:00:03,718] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: say_hi.get_name_task scheduled__2022-06-15T00:00:00+00:00 [queued]>
[2022-06-16 00:00:03,778] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: say_hi.get_name_task scheduled__2022-06-15T00:00:00+00:00 [queued]>
[2022-06-16 00:00:03,779] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 00:00:03,779] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-16 00:00:03,780] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 00:00:03,898] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): get_name_task> on 2022-06-15 00:00:00+00:00
[2022-06-16 00:00:03,917] {standard_task_runner.py:52} INFO - Started process 2300 to run task
[2022-06-16 00:00:03,976] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'say_hi', 'get_name_task', 'scheduled__2022-06-15T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/username.py', '--cfg-path', '/tmp/tmpmgyrxn78', '--error-file', '/tmp/tmphw540a_2']
[2022-06-16 00:00:03,990] {standard_task_runner.py:80} INFO - Job 28: Subtask get_name_task
[2022-06-16 00:00:04,433] {task_command.py:370} INFO - Running <TaskInstance: say_hi.get_name_task scheduled__2022-06-15T00:00:00+00:00 [running]> on host 6a14fd44e8dc
[2022-06-16 00:00:04,945] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=say_hi
AIRFLOW_CTX_TASK_ID=get_name_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-15T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-15T00:00:00+00:00
[2022-06-16 00:00:04,947] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-16 00:00:04,947] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo Alma > /opt/***/dags/username.txt']
[2022-06-16 00:00:05,040] {subprocess.py:85} INFO - Output:
[2022-06-16 00:00:05,057] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-16 00:00:05,365] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=say_hi, task_id=get_name_task, execution_date=20220615T000000, start_date=20220616T000003, end_date=20220616T000005
[2022-06-16 00:00:05,534] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-16 00:00:05,969] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
